src/tests/eventpipe/ThreadPoolEvents.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using EventPipe.UnitTests.Common;
+using Microsoft.Diagnostics.Tracing;
+
+namespace EventPipe.UnitTests.ThreadPoolValidation
+{
+    public class ProviderTests
+    {
+        private readonly ITestOutputHelper output;
+
+        public ProviderTests(ITestOutputHelper outputHelper)
+        {
+            output = outputHelper;
+        }
+
+        [Fact]
+        public async void ThreadPool_ProducesEvents()
+        {
+            await RemoteTestExecutorHelper.RunTestCaseAsync(() => 
+            {
+                Dictionary<string, ExpectedEventCount> _expectedEventCounts = new Dictionary<string, ExpectedEventCount>()
+                {
+                    { "Microsoft-Windows-DotNETRuntime", -1 },
+                    { "Microsoft-Windows-DotNETRuntimeRundown", -1 }
+                };
+
+                var providers = new List<Provider>()
+                {
+                    //ThreadingKeyword (0x10000): 0b10000_0000_0000_0000
+                    new Provider("Microsoft-Windows-DotNETRuntime", 0b10000_0000_0000_0000, EventLevel.Informational)
+                };
+
+                Action _eventGeneratingAction = () => 
+                {
+                    Task[] taskArray = new Task[1000];
+                    for (int i = 0; i < 1000; i++)
+                    {
+                        if (i % 10 == 0)
+                            Logger.logger.Log($"Create new task {i} times...");
+                        Task task = new Task(() => TestTask());
+                        taskArray[i] = task;
+                        taskArray[i].Start();
+                    }
+                    Task.WaitAll(taskArray);
+                };
+
+                void TestTask()
+                {
+                    Thread.Sleep(100);
+                }
+
+                Func<EventPipeEventSource, Func<int>> _DoesTraceContainEvents = (source) => 
+                {
+                    int ThreadStartEvents = 0;
+                    //int ThreadStopEvents = 0;
```

Comment:
Delete these lines rather than commenting them out.

Reply:
Fixed: deleted the lines.